packages/dcos-integration-test/extra/test_misc.py

```diff
@@ -17,9 +17,6 @@ def test_load_user_config():
 
 
 def test_load_expanded_config():
-    with open("/opt/mesosphere/etc/expanded.config.json", "r") as f:
-        expanded_config = json.load(f)
-
```

Comment:
This test's name is a little misleading now that it's no longer actually loading the config. Removing "load" from the name would help clarify.